events-api/src/modules/events/events.service.ts

```diff
@@ -1,10 +1,26 @@
 import { EventModel } from "#/modules/events/events.model";
 import { EventsRepository } from "#/modules/events/events.repository";
+import { EventCreateDto } from "./dto/requests/events-create.dto";
 
 export class EventsService {
   static async getEvents(): Promise<EventModel[]> {
     const events = await EventsRepository.getAll();
 
     return events;
   }
+
+  static async createEvent(newEvent: EventCreateDto): Promise<void> {
+    await EventsRepository.create(newEvent);
+  }
+
+  static async updateEvent(
+    event: EventCreateDto,
+    id: string
+  ): Promise<null | unknown> {
+    return await EventsRepository.update(event, id);
```

Comment:
This method should return the updated event data

Reply:
![image](https://github.com/user-attachments/assets/f0aa9a6d-a3d8-4840-a8d2-6502baf973bc)

Reply:
Again, you can just select a record after updating. This limitation is not a blocker

Reply:
I have changed status code to 200 because of postman not showing res content
![image](https://github.com/user-attachments/assets/5b34ee41-e231-4aa8-b100-75a81937a697)